godotproject/PlayerTongue.gd

```diff
@@ -70,8 +70,6 @@ func _process(delta):
 func _on_Player_tongue_start(facing):
 	if idle:
 		start_shoot(facing)
-	pass # Replace with function body.
-
```

Comment:
Are we ok to submit cleanups like this in otherwise unrelated changes? I think it's probably good bc that way you don't have to make a whole other change and PR for a tiny thing like this.

Reply:
I agree, I think it should be good. I generally try to clean up small stuff like one liner deletions or old comments as I find them.